go/cmd/dolt/commands/sql.go

```diff
@@ -183,6 +191,74 @@ func (cmd SqlCmd) Exec(ctx context.Context, commandStr string, args []string, dE
 		return HandleVErrAndExitCode(verr, usage)
 	}
 
+	var cfgDirPath string
```

Comment:
You should extract all this logic you added into a new method called `getConfigFilesLocations`